chapters/ch20.md

```diff
@@ -1,49 +1,57 @@
 ->
-
+Nếu tỉ lệ lỗi trên tập huấn luyện là 15% (tức độ chính xác 85%), nhưng mục tiêu của bạn là 5% lỗi (độ chính xác 95%), thì vấn đề trước tiên cần giải quyết là cải thiện hiệu quả thuật toán của bạn trên tập huấn luyện. Hiệu quả trên tập phát triển/kiểm tra thường thấp hơn hiệu quả trên tập huấn luyện. Bởi vậy nếu bạn đang có độ chính xác 85% trên các mẫu mà thuật toán từng thấy, không có cách nào để đạt được độ chính xác 95% cho các mẫu mà thuật toán chưa thấy bao giờ.
 
 Suppose as above that your algorithm has 16% error (84% accuracy) on the dev set. We break the 16% error into two components:
 ->
-
+Giả sử như trên rằng thuật toán của bạn có 16% lỗi (độ chính xác 84%) trên tập phát triển. Chúng ta tách 16% lỗi này ra hai thành phần:
 
 * First, the algorithm’s error rate on the training set. In this example, it is 15%. We think of this informally as the algorithm’s **​bias​**.
 ->
+* Thứ nhất, tỉ lệ lỗi của thuật toán trên tập huấn luyện, là 15% trong ví dụ này. Chúng ta tạm nghĩ giá trị này như **độ chệch** của thuật toán.
 
 
 * Second, how much worse the algorithm does on the dev (or test) set than the training set. In this example, it does 1% worse on the dev set than the training set. We think of this informally as the algorithm’s **​variance** [6]
 ->
+Thứ hai, hiệu quả của thuật toán trên tập phát triển (hoặc kiểm tra) kém hơn bao nhiêu so với trên tập huấn luyện. Trong ví dụ này, thuật toán làm việc kém hơn 1% trên tập phát triển so với tập huấn luyện. Chúng ta tạm coi giá trị này như **phương sai** của thuật toán [6].
 
 
 Some changes to a learning algorithm can address the first component of error—​**bias​**—and improve its performance on the training set. Some changes address the second component—​**variance​**—and help it generalize better from the training set to the dev/test sets [7]. To select the most promising changes, it is incredibly useful to understand which of these two components of error is more pressing to address.
 ->
-
+Một số thay đổi trong thuật toán học có thể giải quyết thành phần thứ nhất của lỗi -- **độ chệch** -- và cải thiện hiệu quả của nó trên tập huấn luyện. Một số thay đổi giải quyết thành phần thứ hai -- **phương sai** -- và giúp thuật toán tổng quát hóa tốt hơn từ tập huấn luyện tới tập phát triển/kiểm tra [7]. Để lựa chọn thay đổi tiềm năng nhất, sẽ rất hữu ích khi hiểu thành nào trong hai thành phần lỗi là đáng để giải quyết hơn.
 
 Developing good intuition about Bias and Variance will help you choose effective changes for your algorithm.
 ->
-
+Phát triển một trực giác tốt về Độ chệch và Phương sai sẽ giúp bạn có những thay đổi hữu hiệu cho thuật toán.
```

Comment:
"Phát triển một trực giác tốt về Độ chệch và Phương sai sẽ giúp bạn **có** những thay đổi hữu hiệu cho thuật toán.
" ==> "Phát triển một trực giác tốt về Độ chệch và Phương sai sẽ giúp bạn **chọn** những thay đổi hữu hiệu cho thuật toán." Em thích từ chọn hơn vì nó có nghĩa là mình vẫn phải ra quyết định.